tests/system/test_asyncpg_connection.py

```diff
@@ -87,7 +90,7 @@ async def create_sqlalchemy_engine(
             user=user,
             password=password,
             db=db,
-            ip_type="public",  # can also be "private" or "psc"
+            ip_type=ip_type,
```

Comment:
nit: instead of this comment being during test case, it should be here so that it shows up in the docs
```suggestion
            ip_type=ip_type,  # can be "public", "private" or "psc"
```